src/test/scala/nl.knaw.dans.easy.authinfo/TestSupportFixture.scala

```diff
@@ -67,12 +67,18 @@ trait TestSupportFixture extends FlatSpec with Matchers with Inside with BeforeA
         addProperty("bag-store.read-timeout-ms", "1")
         addProperty("solr.url", "http://hostThatDoesNotExist")
         addProperty("solr.collection", "authinfo")
-      })
+      },
+        licenses = new Licenses(new PropertiesConfiguration() {
```

Comment:
I think the formatting is a bit off here...

Reply:
No, this is the outcome of `Reformat Code` in the IDE